go/billing/tests/test_admin.py

```diff
@@ -72,3 +75,20 @@ def test_account_admin_view(self):
         self.assertContains(response, 'Description')
         self.assertContains(response, 'Credit balance')
         self.assertContains(response, 'Last topup balance')
+        self.assertContains(response, 'Is developer')
+
```

Comment:
Could we perhaps have an additional separate test for ``is_developer``? I know it's tested by the two new tests, but since it's used separately in the list view it feels like it could do with its own.